src/ApprenticePortal/SFA.DAS.ApprenticePortal/Application/Commands/ApprenticePatchCommand.cs

```diff
@@ -0,0 +1,13 @@
+﻿using MediatR;
+using Microsoft.AspNetCore.JsonPatch;
+using SFA.DAS.ApprenticePortal.Models;
+using System;
+
+namespace SFA.DAS.ApprenticePortal.Application.Commands.ApprenticePatch
+{
+    public class ApprenticePatchCommand : IRequest<Unit>
```

Comment:
IRequest<Unit> isn't required, IRequest should be fine

Reply:
Sorry 

IRequest<Unit>

is not usually defined explicitly

IRequest (without the Unit) is fine

Reply:
but I'm fine with it either way